ee/clickhouse/queries/funnels/funnel_time_to_convert.py

```diff
@@ -60,10 +60,19 @@ def get_query(self) -> str:
         ]
         steps_average_conversion_time_expression_sum = " + ".join(steps_average_conversion_time_identifiers)
 
+        steps_average_conditional_for_invalid_values = [
+            f"{identifier} >= 0" for identifier in steps_average_conversion_time_identifiers
+        ]
+        # this is protection against the CH bug: https://github.com/ClickHouse/ClickHouse/issues/26580
```

Comment:
Nit: Suggestion on comment style:

```
# :HACK: Protect against CH bug https://github.com/ClickHouse/ClickHouse/issues/26580
#   once the issue is resolved, stop skipping the test: test_auto_bin_count_single_step_duplicate_events
#   and remove this comment
```

I use the following meta-comments: `:TRICKY:` `:TODO:`. They are easier to grep for and convey reader should look out clearer than free-form text.